fiftyone/server/routes/get_similar_labels_frames.py

```
@@ -0,0 +1,297 @@
+"""
+
+from bson import ObjectId
+from starlette.endpoints import HTTPEndpoint
+from starlette.requests import Request
+from starlette.responses import JSONResponse
+
+import fiftyone.core.labels as fol
+import fiftyone.core.odm as foo
+import fiftyone.core.view as fov
+import fiftyone.server.view as fosv
+from fiftyone.core.expressions import ViewField as F
```

Ruff (0.8.2)

`fiftyone.core.expressions.ViewField` imported but unused

Remove unused import: `fiftyone.core.expressions.ViewField`

(F401)